Tests/Test_CriticalState_2.c

```diff
@@ -0,0 +1,127 @@
+/* Copyright (c) 2021 UT Longhorn Racing Solar */
+
+#include "common.h"
+#include "config.h"
+#include "os.h"
+#include "Tasks.h"
+#include "stm32f4xx.h"
+#include "BSP_Contactor.h"
+#include "BSP_Lights.h"
+#include "BSP_PLL.h"
+
+OS_TCB Task1_TCB;
+CPU_STK Task1_Stk[256];
+OS_TCB Task2_TCB;
+CPU_STK Task2_Stk[256];
+
+/******************************************************************************
+ * Critical State Task Test Plan
+ * 
+ * 1. Build the BPS code for the stm32f413 for this test file (see README.md)
+ * 2. Flash the BPS (see README.md)
+ * 3. Connect the BPS UART USB to your laptop and open PuTTY (or comparable)
+ * 4. Connect the BPS test board
```

Comment:
I don't think you need either of these steps, since you are not using UART or Voltage measurements in this test

Reply:
Fixed

Reply:
I still see the instructions to set up UART and the Test Board, and I still don't see where you use them in the test

Reply:
Putty/UART are only useful if you actually use UART in your code. You are not using it anywhere, so I don't understand why step 3 of the instructions is to connect UART and open Putty. 

The BPS test board is used for providing set voltage inputs to the BPS. You are not reading any voltages (or any other external inputs) in your code, so I don't see why step 4 is to connect the BPS test board